### ISSUE TITLE
Cant Install 

### ISSUE THREAD

**Issue by nileshbadgi** [socialigniter/socialigniter#70]
Cant install the package 
Showing permission error but i have 777 permission.
![install social-igniter 2014-08-29 13-22-23](https://cloud.githubusercontent.com/assets/5346707/4087698/377b818c-2f52-11e4-9b5c-fdf38d1ae167.png)

**Comment by phatyh** [comment]
create this file

**Comment by straweb** [comment]
can't install event after creation of config files..

error in migration of 002 migration file duplication of status in categories table

**Comment by rhisss** [comment]
Installation steps are not clear for non-geeks
1. Do we download both https://github.com/socialigniter/socialigniter/archive/master.zip and https://github.com/socialigniter/system ?
2. I am testing in localhost and I cannot understand what this means:'Make sure /system directory lives at the same folder level as the your app (core Social-Igniter)' 
3. Can someone please clarify/give step by step how-to-install

Please make it as easy as installing Drupal/Wordpress etc for example
http://localhost/wordpress  - you get a nice install screen without having to do anything before
http://localhost/socialiginter - cannot it be that easy please?

**Comment by bnvk** [comment]
@rhisss the goal was to make SI as easy as Drupal/Wordpress... however my client who had been funding dev and I parted ways. And for one reason or another (PHP, CodeIgniter, my explanation etc...) I was never able to get a "community" of contributors to help. Thus, this project is close to abandoned at the moment. 

That said, I still use SI for my personal site and if there was one or two steady contributors, I'd be inspired to work on it again ;)

**Comment by otengkwame** [comment]
I think i'm interested in SI. You know it can become something big in future. I would like to join the community that will bring it back to life. I had issues with installing at first so i had to drop it. But the truth is i borrowed some codes to write some models and controllers. Let's make it happen

**Comment by ebredy** [comment]
I would like to contribute as well.  the first step is figuring out what localhost/setup is suppose to call.

**Comment by muzanella11** [comment]
Hi guys, how can I get this database for socialigniter?

**Comment by ebredy** [comment]
the database as actually part of the application/config/install.php as a set of multidimensional arrays.  if you can't install it, a number of things went wrong.

**Comment by cripler** [comment]
while Creating Database... it's  freezes. help me

**Comment by ebredy** [comment]
there is a number of bugs with this social network.  Do you know how to look at your error logs?  Are you installing on a hosting company or a local host?

**Comment by cripler** [comment]
yes it's local host but also try on hosting with cpanel same was there. can you explain me how to see error logs ?

**Comment by ebredy** [comment]
what are you using wampp or xampp for a local environment/host?  Also do you know how to enable php error?

**Comment by cripler** [comment]
i'm using xampp. in php.ini file display_errors=On

**Comment by ebredy** [comment]
click the logs button next to the apache process running.  the logs menu should come up and select the php error logs.

**Comment by cripler** [comment]
![1](https://cloud.githubusercontent.com/assets/13084438/9914675/f91cbb2c-5cc3-11e5-82f3-3392763c282a.PNG)
freezes like this 
![2](https://cloud.githubusercontent.com/assets/13084438/9914676/f9378970-5cc3-11e5-8f1b-8448458bd7bc.PNG)
log is empty

**Comment by ebredy** [comment]
Do you know how to enable on screen errors?

**Comment by ebredy** [comment]
so you need to place this on top of your index.php file:

 error_reporting(E_ALL);
ini_set('display_errors', 1);

**Comment by ebredy** [comment]
another thing this social network has a service architecture so you will need to view your network traffic when by enabling developer tools and looking at the post/get data and response from jquery to server and back.  if there is an error the response will respond with an error.  an addition, I can help you by sending you some of my code that I have or doing a remote desktop session using team viewer.

**Comment by cripler** [comment]
@ebredy  if you can remote my pc and fix it that will be perfect thank you.

**Comment by otengkwame** [comment]
@ebredy please, have you been able to install the social network?

**Comment by ebredy** [comment]
yes I have installed it.

**Comment by arjunrajmr** [comment]
Can anybody submit the entire database schema

**Comment by harishningegowda** [comment]
Can someone update it to php7

**Comment by harishningegowda** [comment]
I am trying to install this app on codeigniter 3, I only got close to hack the installation code and create tables. The process is not really smooth, and should have used codeigniter itself for installation.

**Comment by bnvk** [comment]
@harishningegowda yep, that is to be expected. If anyone wants to help take over this project, send pull requests, or be added as a maintainer, i'm happy to discuss doing so!

**Comment by otengkwame** [comment]
@ebredy, please can you in any way, package a working version of this app and mail it to me?. I will be very grateful. mail: developerkwame@gmail.com
